FRESH_START_AND_RESTART.s

```diff
@@ -1000,7 +1000,7 @@ RENDN00		CS	MMNUMBER
 		EXTEND
 		BZF	KILL20	
 		
-		CA	FLAGWRD0		# IS RENDZVOO FLAG SET
+		CA	FLAGWRD0		# IS RENDEZVOUS FLAG SET
```

Comment:
Neither of these are correct.

Actual word in source is `RENDEZVOO` [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0201.jpg)